website/src/5.x/api/utils/get-js-transform-rules.md

```diff
@@ -0,0 +1,131 @@
+# getJsTransformRules
+
+A helper function that generates Rspack `module.rules` configuration for transforming JavaScript, TypeScript, and Flow files. It combines SWC loader configuration for JS/TS files with Flow and codegen transformations. You can consider it an equivalent of `@react-native/babel-preset`, but for SWC.
```

Comment:
could mention that this is combining `getFlowTransformRules`, `getCodegenTransformRules` and `getSwcLoaderOptions` and you should use this instead of each individually. Similarly, in said module rules helpers you could add a note that you shouldn't need to use these, and use `getJsTransformRules` instead